docs/configuration.rst

```diff
@@ -142,6 +142,23 @@ PAPERLESS_SECRET_KEY=<key>
 
     Default is listed in the file ``src/paperless/settings.py``.
 
+PAPERLESS_URL<url>
```

Comment:
Several of these environment vars are missing the `=` (including some existing ones).  Make it consistent?